lexicons/com/atproto/server/createAccount.json

```diff
@@ -37,7 +38,9 @@
         {"name": "InvalidPassword"},
         {"name": "InvalidInviteCode"},
         {"name": "HandleNotAvailable"},
-        {"name": "UnsupportedDomain"}
+        {"name": "UnsupportedDomain"},
+        {"name": "CouldNotResolveDid"},
+        {"name": "PoorlyFormattedDidDoc"}
```

Comment:
Sorry for the total nit, but I'm not feeling 100% sure about this error name.  My understanding is that this is used when there's a valid did doc but it's not compatible with the PDS that the account is being created on.  What if it were something like `IncompatibleDidDoc`?  Promise I'm not going to get hung-up on this if you prefer it as-is 😆